CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 1.8.0
+## General
+Added
+* Add EBS CSI driver policy to nodegroup IAM role for integration tests ([#791])
+
+Fixed
+* Replace ring cryptography library with aws-lc-rs ([#795])
+* Limit permissions of GITHUB_TOKEN in CI workflow ([#787])
```

Comment:
Probably better in Misc, but also not really worth calling it out if it wasn't a contribution